packages/design-system/src/TextInput/TextInput.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+
+import TextInput from "./index";
+
+export default {
+  title: "Design System/TextInput",
+  component: TextInput,
+} as ComponentMeta<typeof TextInput>;
+
+// eslint-disable-next-line react/function-component-definition
+const Template: ComponentStory<typeof TextInput> = (args) => {
+  return <TextInput {...args} />;
+};
+
+export const TextInputExample = Template.bind({});
+TextInputExample.storyName = "Text Input";
+TextInputExample.args = {
+  $padding: "8px 14px",
```

Comment:
$?

Reply:
This is from the component itself. I'm guessing whoever wrote this wrote the variable names like this to distinguish them from some other kind of padding which is also passed as a prop. I agree we should change it, but we should do that in V2.
https://github.com/appsmithorg/design-system/blob/b729176a93028a22d36accc45729bfe17cb67d0b/packages/design-system/src/TextInput/index.tsx#L125